src/hooks/useNotificationTypes.tsx

```diff
@@ -288,6 +289,27 @@ export const notificationTypes: NotificationTypeConfig[] = [
       const currentDate = new Date();
 
       useEffect(() => {
+        if (currentDate <= twitter200BVolumeExpirationDate) {
+          trigger(
+            ReleaseUpdateNotificationIds.Twitter200BVolume,
+            {
+              icon: <AssetIcon symbol={chainTokenLabel} />,
+              title: stringGetter({
+                key: 'NOTIFICATIONS.TWITTER_DYDX_200B_GIVEAWAY.TITLE',
+              }),
+              body: stringGetter({
+                key: 'NOTIFICATIONS.TWITTER_DYDX_200B_GIVEAWAY.BODY',
+                params: {
+                  HERE_LINK: 'https://x.com/dYdX/status/1819342483794415784',
```

Comment:
The HERE_LINK would need to to be an Link component, or else it will just end up rendering the link as text. See: https://github.com/dydxprotocol/v4-web/blob/main/src/hooks/useNotificationTypes.tsx#L326